tests/playwright/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "bootc-tests-playwright",
+  "version": "0.0.1",
+  "description": "Podman Desktop BootC extension Playwright E2E tests",
+  "scripts": {
+    "test:e2e:setup": "xvfb-maybe --auto-servernum --server-args='-screen 0 1280x960x24' --",
+    "test:e2e": "npm run test:e2e:setup vitest run src/ --pool=threads --poolOptions.threads.singleThread --poolOptions.threads.isolate --no-file-parallelism"
+  },
+  "author": "Red Hat",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@podman-desktop/tests-playwright": "^0.0.202403202143-15b0f59"
+  },
+  "devDependencies": {
+    "@playwright/test": "^1.42.1",
+    "@podman-desktop/tests-playwright": "^0.0.202403202143-15b0f59",
+    "@types/node": "^20",
+    "electron": "^29.1.4",
```

Comment:
not a blocker for this PR, but here probably we should add that as a dependency on `@podman-desktop/tests-playwright"` side so we don't need to enforce that there

Reply:
Just `electron` or all?

Reply:
I would say for all dependencies that we're not using from this project

for example when writing a spec.ts file you're using

```
import type { Locator, Page } from '@playwright/test';
import { afterAll, beforeAll, test, describe, beforeEach, expect } from 'vitest';
```

so anyway you need to depend on playwright and vitest so these one are kind of mandatory
but for electron as we don't have direct import on it, it should be transitive

Reply:
I see.  That makes perfect sense.thx!